syft/pkg/cataloger/binary/cataloger_test.go

```diff
@@ -954,7 +954,7 @@ func assertPackagesAreEqual(t *testing.T, expected pkg.Package, p pkg.Package) {
 					return l.ToSlice()
 				}),
 				cmpopts.IgnoreUnexported(pkg.Package{}, file.Location{}),
-				cmpopts.IgnoreFields(pkg.Package{}, "CPEs", "FoundBy", "MetadataType", "Type"),
+				cmpopts.IgnoreFields(pkg.Package{}, "CPEs", "FoundBy", "Metadata", "Type", "Locations", "Licenses"),
```

Comment:
question for reviewer: does it make sense to try and reuse some of the test code from here https://github.com/anchore/syft/blob/main/syft/pkg/cataloger/internal/pkgtest/test_generic_parser.go

Reply:
side notes:
- metadata cannot be ignored
- locations should probably not be ignored